plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/payment-method-card.tsx

```diff
@@ -34,21 +36,42 @@ const PaymentMethodCard = ( {
 	showSaveOption,
 }: PaymentMethodCardProps ) => {
 	const { isEditor } = useEditorContext();
-	const { shouldSavePaymentMethod, customerId } = useSelect( ( select ) => {
-		const paymentMethodStore = select( paymentStore );
-		const checkoutStore = select( checkoutStoreDescriptor );
-		return {
-			shouldSavePaymentMethod:
-				paymentMethodStore.getShouldSavePaymentMethod(),
-			customerId: checkoutStore.getCustomerId(),
-		};
-	} );
+	const { shouldSavePaymentMethod, customerId, createAccount } = useSelect(
```

Comment:
`useSelect` is typed to require two arguments, I know it works properly without providing a second argument, but the return values are all typed as `any` when used like that. If you provide second argument as `[]` the return values are properly typed:
```
{
  shouldSavePaymentMethod: boolean;
  customerId: number;
  createAccount: boolean;
}
```
After looking at the type of `createAccount`, it would be clearer to rename it to `shouldCreateAccount`. What do you think?